test/testclangimport.cpp

```diff
@@ -1015,7 +1015,7 @@ class TestClangImport: public TestFixture {
         GET_SYMBOL_DB(clang);
 
         // Enum scope and type
-        ASSERT_EQUALS(3, db->scopeList.size());
+        ASSERT_EQUALS(3UL, db->scopeList.size());
```

Comment:
The tool is too stupid here in my opinion. The original code looks fine as far as I see.. what possible real danger could there be.

Reply:
This is just so the `assertEquals()` can be replaced with a template later on. That requires the same types on both sides. The code is completely inconsistent and this just offers some consistency. This doesn't fix any UBSAN warnings yet. Those only happen with `long long` and `unsigned long long` are involved and are still present.

Reply:
Imho I think the original code is nicer. Template limitations shouldn't be worked around in our test code.